casacore/tables/table.py

```diff
@@ -1796,19 +1795,18 @@ def browse(self, wait=True, tempname="/tmp/seltable"):
                 os.system('casabrowser ' + self.name() + waitstr1)
             elif len(tempname) > 0:
                 six.print_("  making a persistent copy in table " + tempname)
-                tx = self.copy(tempname)
-                tx = 0
```

Comment:
`self.copy` has side effects (namely it creates a new file). So please do not remove the line. I think replacing these two lines with just
`self.copy(tempname)` would be ok, since indeed the return value is not used.